public/index.php

```diff
@@ -33,14 +52,55 @@
                             echo "text-danger";
                         }
                         ?>"><?php echo (float) $monthTotal; ?> <i class="fa fa-money" aria-hidden="true"></i>
-                    </p>
+                    </p>                        
+                </div>
+            </div>
+            <div class="legend"></div>
                 
-         </section>
-
+        </section>
         <section class='footer'>
                     <?php include public_path("layouts/footer.php") ?>
         </section>
-
     </div>
+
+    <script>
```

Comment:
Please, move this script to its own script asset file in a directory for pages like Scss pages for example. And wrap the code in an IIFE function.

Reply:
All CSS animation will be changed using anime.js ( js animation library ) to avoid all native CSS into a more efficient js script